sqlalchemy_continuum/unit_of_work.py

```diff
@@ -334,3 +335,4 @@ def assign_attributes(self, parent_obj, version_obj):
             except sa.orm.exc.ObjectDeletedError:
                 value = None
             setattr(version_obj, prop.key, value)
+            flag_modified(version_obj, prop.key)
```

Comment:
I'm confused, I probably missed an episode. I thought you wanted to modify `invenio-records` to use a `MutableDict` on the `json` column instead?

Reply:
Discussed on zulip https://inspirehep.zulipchat.com/#narrow/stream/122209-reviews/topic/sqlalchemy-continuum.20fork/near/228236408